README.md

```diff
@@ -21,6 +21,44 @@ Muchos is structured into two high level components:
 
 Checkout [Uno] for setting up Accumulo or Fluo on a single machine.
 
+## Requirements
+
+Muchos requires the following:
+
+* Python 2
+* [awscli] & [boto3] libraries - Install using `pip install awscli boto3 --upgrade --user`
+* `~/.aws` [configured][aws-config] on your machine. Can be done using [aws configure][awscli-config].
+* `ssh-agent` installed and running
+
+## Quickstart
+
+The following commands will install Muchos, launch an EC2 cluster, and setup/run Accumulo:
+
+```bash
+git clone https://github.com/apache/fluo-muchos.git
```

Comment:
This info is redundant with Installation section.

Reply:
Fixed